root/io/io/CMakeLists.txt

```diff
@@ -4,3 +4,5 @@ RB_ADD_GBENCHMARK(IOBenchmarks
   Core RIO Tree TreePlayer
   )
  
+ add_test(NAME rootbench-IOBenchmarks
+		COMMAND IOBenchmarks --benchmark_format=json)
```

Comment:
We probably want `--benchmark_colors=none`, otherwise the output is a little garbled.

Reply:
Sorry, I mistaken it should be --benchmark_out_format=json (for Telegram agent to be submitted into InfluxDB)